crates/kitsune_p2p/kitsune_p2p/src/spawn/actor/test_util.rs

```diff
@@ -1,5 +1,11 @@
+// TODO: not sure how to fix these unused warnings, something to do with
```

Comment:
Yeah I think they're used when you compile for test but the compiler doesn't realise that and sees them for some reason when compiling other builds... or something like that. Also confused by this but good to get rid of the warnings until we can figure it out